Wikipedia/Code/YearInReviewCoordinator.swift

```diff
@@ -266,69 +286,81 @@ final class YearInReviewCoordinator: NSObject, Coordinator {
         return String.localizedStringWithFormat(format)
     }
-    
+
     func personalizedSlide3Overlay(editCount: Int) -> String {
         guard editCount < 500 else {
             return "500+"
         }
-        
+
         let formatter = NumberFormatter()
         formatter.numberStyle = .decimal
         let number = NSNumber(value: editCount)
         return formatter.string(from: number) ?? String(editCount)
     }
-    
+
     var personalizedSlide5Title: String {
         return WMFLocalizedString("year-in-review-personalized-donate-title", value: "Thank you for your contribution!", comment: "Year in review, personalized donate slide title for users that donated at least once that year. ")
     }
-    
+
     func personalizedSlide5Subtitle(languageCode: String?) -> String {
-        
+
         let urlString: String
         if let languageCode {
             urlString = "https://www.mediawiki.org/wiki/Wikimedia_Apps/About_the_Wikimedia_Foundation/\(languageCode)"
         } else {
             urlString = "https://www.mediawiki.org/wiki/Wikimedia_Apps/About_the_Wikimedia_Foundation"
         }
-        
+
         let format = WMFLocalizedString("year-in-review-personalized-donate-subtitle", value: "Thank you for supporting Wikipedia and a world where knowledge is free for everyone. Every single edit and donation helps improve people’s access to accurate and reliable information, especially in a rapidly changing world. [Learn more about our work](%1$@).", comment: "Year in review, personalized donate slide subtitle for users that donated at least once that year. %1$@ is replaced with a MediaWiki url with more information about WMF. Do not alter markdown when translating.")
         return String.localizedStringWithFormat(format, urlString)
     }
-    
+
+    func personalizedSavedCountSlideTitle(savedCount: String) -> String {
+        let format = WMFLocalizedString("year-in-review-personalized-saved-title-format", value: "You saved %1$@ articles", comment: "Year in review, personalized saved articles slide subtitle. %1$@ is replaced with the number of articles the user saved.")
+        return String.localizedStringWithFormat(format, savedCount)
+    }
+
+    func personalizedSavedCountSlideSubtitle(savedCount: String, articleNames: [String]) -> String {
+        let format = WMFLocalizedString("year-in-review-personalized-saved-subtitle-format", value: "You saved %1$@ articles this year, including \"%2$@\", \"%3$@\" and \"%4$@\". Each ", comment: "Year in review, personalized saved articles slide subtitle. %1$@ is replaced with the number of articles the user saved, %2$@, %3$@ and %4$@ are replaced with the names  three random articles the user saved.")
```

Comment:
Also maybe consider a plural format. What if they have less than 3 saved articles, is this shown? How should the article titles act?

Update: it does look like we only display this slide if 3 or more articles are saved. For safety maybe we should be strict about what we pass in via the parameters (so an article1Title, article2Title, and article3Title param). That way there's no risk of a future dev passing in a short articleNames array and causing an out of index crash.